### ISSUE TITLE
Endpoint to let users know which version of the server they are using.

### ISSUE THREAD

**Issue by vtomole** [SupertechLabs/general-superstaq#211]
### What is happening?

Right now users can do

```python
import general_superstaq as gss
import qiskit_superstaq as qss
import cirq_superstaq as css

print(gss.__version__))
print(qss.__version__))
print(css.__version__))
```

But they can't tell which version of the server they are using.

This issue is to add an endpoint to SuperstaQ and add the appropriate methods in the clients that will display the SuperstaQ version.

### What are the current alternatives?

Right now it's impossible for clients to know this.

### Any additional context?
@SalahedeenIssa , @richrines1  and I believe this will help external users when they run into problems with SuperstaQ

**Comment by stephanielee9** [comment]
For Bharath when he starts

**Comment by vtomole** [comment]
@bharat-thotakura 
A good UX for this would be 

```python
import general_superstaq as gss
print(gss.__ss_version__)
```

**Comment by richrines1** [comment]
> ```python
> import general_superstaq as gss
> print(gss.__ss_version__)
> ```

it would have to be an api call right? so would be a call from a client, e.g. something like `css.Service().superstaq_version()` or `qss.SuperstaQProvider.superstaq_version()` (where `superstaq_version()` would be implemented in `gss.UserConfig` (or whatever we change it's name to))

**Comment by vtomole** [comment]
Yes. I was thinking that `gss.__ss_version__` could alias the client call to make it match how you check version of the open source packages. But haven't thought about it too deeply. Could be an awful idea.

**Comment by richrines1** [comment]
i think the problem would be that a client hasn't been instantiated yet, so it would break if your api key isn't set as an env variable/etc (and in general it'd be weird if just importing gss made a server request)

**Comment by vtomole** [comment]
Agree.

**Comment by richrines1** [comment]
but another thought after conversation with @paaige earlier: we could also return the superstaq version as part of the metadata for any other api call. then we could even save it as an attribute of the client

**Comment by vtomole** [comment]
That sounds good to me.

**Comment by bharat-thotakura** [comment]
I removed the GetVersion endpoint and put the superstaq version in the metadata. See SupertechLabs/SuperstaQ#2097

**Comment by vtomole** [comment]
Nice! The next step is to add a method in the client https://github.com/SupertechLabs/general-superstaq/blob/65715ea56d2d0042dafd6d49463d2a3f7eefe257/general_superstaq/superstaq_client.py#L27 that retrieves the version from the server and returns it.
